spec/support/schemas/log_entries/create.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$ref": "#/definitions/Root",
+  "definitions": {
+    "Root": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "id": {
+          "type": "integer"
+        },
+        "data": {
+          "type": ["number", "string"]
```

Comment:
Most of the content in these files was automatically generated by pasting example API responses into https://jsonformatter.org/json-to-jsonschema , but in some places (such as here, `"number"` => `["number", "string"]`) I had to manually modify some things.